Assets/Scripts/с#/GetComponentEx6/Switch.cs

```diff
@@ -0,0 +1,17 @@
+using UnityEngine;
+
+public class Switch : MonoBehaviour
+{
+    private Switch _keyOff;
+
+    private void Start()
+    {
+        _keyOff = GetComponent<Switch>();
+    }
+
+    private void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Return))
+            _keyOff.enabled = !_keyOff.enabled;
+    }
+}
```

Comment:
What is this task?

Reply:
------------ I don't understand what needs to be done